Source/WebCore/animation/WebAnimation.cpp

```diff
@@ -1890,10 +1890,10 @@ void WebAnimation::setBindingsRangeStart(TimelineRangeValue&& rangeStartValue)
         return;
 
     auto rangeStart = SingleTimelineRange::parse(WTFMove(rangeStartValue), keyframeEffect->target(), SingleTimelineRange::Type::Start);
-    if (rangeStart == m_timelineRange.start)
+    if (m_specifiedRangeEnd == rangeStart)
```

Comment:
Is that check correct? Did you mean to compare `m_specifiedRangeStart` here rather?

Reply:
Fixed.